src/components/Dashboard/NewsContainer.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import withFetch from '../../utils/withFetch';
+import { compose } from 'recompose';
+import NewsItem from './NewsItem';
+import { WidgetContainer, WidgetHeader, WidgetBody } from './Widget';
+
+const NewsFeed = props => {
+	return (
+		<WidgetContainer width="one-half" name="news-feed">
+			<WidgetHeader title="news feed" />
+			<WidgetBody>
+				<table className="table">
+					<tbody>
+					{
+							Object.keys(props.news.data).map( key => <NewsItem key={key} index={key} post={props.news.data[key]} /> )
+					}
+					</tbody>
+				</table>
+			</WidgetBody>
+		</WidgetContainer>
+	);
+}
+
+const NewsFeedWithData = compose(
+	withFetch( `https://humanmade.com/wp-json/wp/v2/posts?_embed`, {}, 'news' )
```

Comment:
Did we get anywhere with looking at the tags/categories on our blog posts and what to exclude here? No worries at this stage but something to come back to.

Reply:
not yet no